src/python/pants/engine/fs.py

```diff
@@ -29,6 +35,8 @@
 Digest = PyDigest
 FileDigest = PyFileDigest
 Snapshot = PySnapshot
+AddPrefix = PyAddPrefix
+RemovePrefix = PyRemovePrefix
```

Comment:
At some point soon, we might consider dropping the `Py` from the names here, and using relative-imports on the Rust side to avoid collisions where necessary.